bin/wmagent-component-standalone

```diff
@@ -15,18 +15,27 @@ wmagent-component-standalone -c $WMA_CONFIG_FILE -p JobAccountant -e $WMA_ENV_FI
 
 import os
 import sys
+# NOTE: Setting the sys.dont_write_bytecode flag bellow is a precaution to avoid sabotaging
+#       the work of the %autoreload module for our default ipython setup in our agents
+sys.dont_write_bytecode = True
 import logging
 import importlib
 import runpy
 import pkgutil
 import inspect
-import sysconfig
+import subprocess
+from pprint import pprint,pformat
+from distutils.sysconfig import get_python_lib
 from argparse import ArgumentParser, RawDescriptionHelpFormatter
 from WMCore.Configuration import loadConfigurationFile
 from WMCore.ResourceControl.ResourceControl import ResourceControl
+from WMCore.Services.CRIC.CRIC import CRIC
```

Comment:
this is added but not used in this PR, do you need it?

Reply:
well, we need to create an instance of the cric service to suppress this pylint warning. And that was the idea of having those  imported in advance. let me check the proper  way of doing this

Reply:
and done